test/unit_tests/braket/default_simulator/openqasm/test_interpreter.py

```diff
@@ -1377,10 +1378,18 @@ def test_missing_input():
     int[8] doubled;
 
     doubled = in_int * 2;
+    qubit q;
+    rx(doubled) q;
     """
     missing_input = "Missing input variable 'in_int'."
-    with pytest.raises(NameError, match=missing_input):
-        Interpreter().run(qasm)
+    # with pytest.raises(NameError, match=missing_input):
```

Comment:
Can these lines be removed altogether?

Reply:
No, since simulator.run still needs bound values. This change does obfuscate that error message since it now fails somewhere else, so we should definitely fix that before merging

Reply:
Oh sorry, in this test yes. But you also caught another issue, which is what I was replying to